### ISSUE TITLE
How to get class wise accuracy/precision scores in multi-class classification. (CIFAR-10 with MOCO)

### ISSUE THREAD

**Issue by aymuos15** [lightly-ai/lightly#472]
I am running the tutorial for CIFAR-10 with MOCO for this certain dataset with 6 class. How do I print the individual class-wise scores?

**Comment by philippmwirth** [comment]
After finetuning the pre-trained ResNet you can use it like any Pytorch model to get predictions and calculate the accuracy or precision from those.

**Comment by aymuos15** [comment]
```
    def validation_step(self, batch, batch_idx):
        x, y, _ = batch
        y_hat = self.forward(x)
        y_hat = torch.nn.functional.softmax(y_hat, dim=1)
        self.accuracy(y_hat, y)
        self.log('val_acc', self.accuracy.compute(),
                 on_epoch=True, prog_bar=True)
```

This is where the val accuracy is computed, we can exactly extract the y_hat and y values right?

**Comment by philippmwirth** [comment]
Correct, you can compute the class-wise probabilities directly in the validation step.
